src/plugins/discover/public/application/main/components/top_nav/open_options_popover.scss

```diff
@@ -1,5 +1,5 @@
-$dscOptionsPopoverWidth: $euiSizeL * 12;
+$dscOptionsPopoverWidth: $euiSizeL * 14;
```

Comment:
:information_source: I've increased the width a bit, so that the "view mode" on top will actually fit in English language into one line.